### DISCUSSION TITLE
Calculate confidence intervals

### DISCUSSION THREAD

**Discussion by BSanau** [pycaret/pycaret#1884] [Q&A] [answered]
I have checked the documentation and I have not found any function that calculates the confidence interval of a prediction. Is it possible to calculate it?

I am predicting some observations for next year and I have no values to compare and estimate the prediction error.

**Comment by PabloJMoreno** [comment, marked as answer]
Hello, you have the parameter `return_pred_int` at the predict_model() function
You also have the plot_model() function that displays the CI

**Reply by ngupta23** [reply]
Also, note that not all models support prediction intervals. You can read more about it here: https://github.com/pycaret/pycaret/discussions/1814
